paths/job_annotations/index.yaml

```
@@ -0,0 +1,39 @@
+---
+summary: List job annotations
+description: Retrieve a list of annotations for a job.
+operationId: job_annotations/index
```

We usually call this `list`, e.g. https://github.com/phrase/openapi/blob/main/paths/branches/index.yaml#L4

```suggestion
operationId: job_annotations/list
```